### ISSUE TITLE
question: why no videos

### ISSUE THREAD

**Issue by kopp** [blizzard4591/openMittsu#21]
This is more a question than an issue:
I saw on [openmitsu.de](https://www.openmittsu.de/) that videos are currently not supported.  Could you please quickly explain, why that is so -- i.e. whether/how videos are handled differently by Threema?

**Comment by blizzard4591** [comment]
Sure - the reason is a simple one. When I was implementing most of the core functionality, the ID I was using had a feature level that did not support videos (or file transfers) . therefore, I found my self unable to test and implement those features. Since I added the necessary functionality for setting the feature level just two weeks ago (I believe), it is now possible to add video support, it just is not that high on my priority list since I do not use videos myself. 
You would be most welcome, should you decide to add support for videos ;)
